static/sass/_pattern_buttons.scss

```diff
@@ -5,7 +5,7 @@
   }
 
   .p-button--tag {
-    @extend %p-button-light;
+    @extend %p-button-light !optional;
```

Comment:
Not sure how much it is in scope of this PR.

What is this supposed to extend? It doesn't seem like `p-button-light` exists anywhere (not in Vanilla or in ubuntu.com).

As far as I can see this `p-button--tag` is used only on security pages.

<img width="666" alt="image" src="https://github.com/canonical/ubuntu.com/assets/83575/623dc400-088f-48d6-8fcc-afe824502c38">

That almost looks like a broken button style, as it doesn't look like button at all.
Is this intended? Would it be fine to just change this to standard p-button, or p-button--base?
This would allow to get rid of this (possibly broken) custom styling.

Reply:
It was throwing errors without the `!optional` bit whenever I ran the project locally, it was just a quick fix for that. But I'm not sure why that class was made in the first place tbh. I'll make a follow up pr